solr/bin/solr

```diff
@@ -2264,6 +2264,7 @@ function start_solr() {
     # users who don't care about useful error msgs can override in SOLR_OPTS with +OmitStackTraceInFastThrow
     "${SOLR_HOST_ARG[@]}" "-Duser.timezone=$SOLR_TIMEZONE" "-XX:-OmitStackTraceInFastThrow" \
     "-XX:OnOutOfMemoryError=$SOLR_TIP/bin/oom_solr.sh $SOLR_PORT $SOLR_LOGS_DIR" \
+    "-Dsolr.packages.local.dir=$SOLR_TIP/contrib" \
```

Comment:
This should be in the windows `.cmd` as well.

Reply:
Should not the bundled "modules" folder always be available? Is it even necessary to make the local packages dir customizable like this? I mean, if you perhaps have your own inhouse pkg that you would want to load in a "trusted" way, you would either copy it into the "modules" folder or you'd want pkg mgr to resolve some other location **in addition**, not instead of the 1st party pkgs?

PS: Is it time to merge in latest master to cut everything over to "modules"?

Reply:
So I agree that this likely doesn't need to be customizable. But if we intend to allow customization, how about we just provide this as a default when reading in the value? (And also read from both sysProp as well as envVar). That way there's no need to modify the bin/solr or bin/solr.cmd